deploy/jiva-csi.yaml

```diff
@@ -0,0 +1,396 @@
+  - kind: ServiceAccount
+    name: openebs-jiva-csi-node-sa
+    namespace: kube-system
+roleRef:
+  kind: ClusterRole
+  name: openebs-jiva-csi-registrar-role
+  apiGroup: rbac.authorization.k8s.io
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: openebs-jiva-csi-node
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: openebs-jiva-csi-node
+  template:
+    metadata:
+      labels:
+        app: openebs-jiva-csi-node
+        role: openebs-jiva-csi
+    spec:
+      priorityClassName: system-node-critical
+      serviceAccount: openebs-jiva-csi-node-sa
+      hostNetwork: true
+      containers:
+        - name: csi-node-driver-registrar
+          image: quay.io/k8scsi/csi-node-driver-registrar:v1.0.1
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+            - "--kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)"
+          lifecycle:
+            preStop:
+              exec:
+                command: ["/bin/sh", "-c", "rm -rf /registration/jiva.csi.openebs.io /plugin/csi.sock"]
+          env:
+            - name: ADDRESS
+              value: /plugin/csi.sock
+            - name: DRIVER_REG_SOCK_PATH
+              value: /var/lib/kubelet/plugins/jiva.csi.openebs.io/csi.sock
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            - name: NODE_DRIVER
+              value: openebs-jiva-csi
+          volumeMounts:
+            - name: plugin-dir
+              mountPath: /plugin
+            - name: registration-dir
+              mountPath: /registration
+        - name: openebs-jiva-csi-plugin
+          securityContext:
+            privileged: true
+            capabilities:
+              add: ["CAP_MKNOD", "CAP_SYS_ADMIN", "SYS_ADMIN"]
+            allowPrivilegeEscalation: true
+          image: openebs/jiva-csi:ci
+          imagePullPolicy: Always
+          args:
+            - "--name=jiva.csi.openebs.io"
+            - "--nodeid=$(OPENEBS_NODEID)"
+            - "--endpoint=$(OPENEBS_CSI_ENDPOINT)"
+            - "--plugin=$(OPENEBS_NODE_DRIVER)"
+            # enableiscsidebug is used to enable debug logs for iscsi operations
+            - "--enableiscsidebug=true"
+            # logging level for klog library used in k8s packages
+            #- "--v=5"
+          env:
+            - name: OPENEBS_NODE_ID
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            - name: OPENEBS_CSI_ENDPOINT
+              value: unix:///plugin/csi.sock
+            - name: OPENEBS_NODE_DRIVER
+              value: node
+            - name: OPENEBS_NAMESPACE
+              value: openebs
+          volumeMounts:
+            - name: plugin-dir
+              mountPath: /plugin
+            - name: device-dir
+              mountPath: /dev
+            - name: pods-mount-dir
+              mountPath: /var/lib/kubelet/pods
```

Comment:
We need to share the path `/var/lib/kubelet` since staging is done at the path `/var/lib/kubelet/plugins` while publish is done on the path `/var/lib/kubelet/pods`

Reply:
This path is still not updated